examples/include-gup/all.gup

```diff
@@ -0,0 +1,4 @@
+#!bash -eu
+echo RUNNING: "$0" "$@"
+
+gup -u main.o
```

Comment:
A bunch of these files have no trailing newline for the file. Generally, my preference (and tools like git/vim etc) is to have such a trailing newline for text files. However, ultimately I'm ambivalent either way, and happy to follow your style for `gup` files.